CONTRIBUTING.md

```diff
@@ -59,6 +59,12 @@ Changes should conform to:
 
 With the exception that 4-space indendation be used.
 
+#### General guidelines for Swift for TensorFlow API docs contribution
+
+* Closely mirror Swift for TensorFlow API docs with implementations in [tensorflow/tensorflow/python](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python) to maintain consistency across platforms.
```

Comment:
```suggestion
* For APIs ported from Python TensorFlow, adopt the API documentation from [tensorflow.org/api_docs](https://www.tensorflow.org/api_docs) as a starting point.
```

I think "adopting as a starting point" is more appropriate than "closely mirroring". It's a goal to write idiomatic Swift documentation, following rules like #160.

Reply:
Done